src/qml/qgismobileapp.qml

```diff
@@ -931,16 +931,35 @@ ApplicationWindow {
     }
   }
 
+  NavigatingDrawer{
+    id: navigatingDrawer
+    openRequested: (navigation.isActive || positioningSettings.showPositionInformation ) &&
+                !elevationProfile.visible &&
+                mapCanvasMap.isEnabled &&
+                !messageLog.visible
+
+    navigation: navigation
+    positioningSettings: positioningSettings
+    positioningPreciseViewHeight: Math.min(mainWindow.height / 2.5, 400)
+    positioningPreciseEnabled: !elevationProfile.visible
+                               && !isNaN(navigation.distance)
+                               && navigation.isActive
+                               && (positioningSettings.alwaysShowPreciseView
+                                  || ( positioningPreciseView.hasAcceptableAccuracy
+                                  &&  positioningPreciseView.projectDistance < positioningPreciseView.precision ))
+
+    positioningInformationViewEnabled: positioningSettings.showPositionInformation && !elevationProfile.visible
+    positionSource: positionSource
+    antennaHeight: positioningSettings.antennaHeightActivated ? positioningSettings.antennaHeight : NaN
```

Comment:
Could we move that logic into the ~~Navigating~~InformationDrawer item itself? Less code in qgismobileapp.qml is better.

On that note, very happy to see you've moved individual information items into the drawer QML file, shrinks our mammoth main qml file a bit :)

Reply:
Thank you for your kind words ♡

actually 'elevationProfile.visible'  is a bit annoying. i can pass it as a property to "InformationDrawer" but I'm struggling to come up with a suitable property name

Reply:
what about `uiConflictFree: !elevationProfile.visible` ?

Reply:
@mohsenD98 , let's just _"cheat"_ for now and just use the globally referenced elevationProfile item from within InformationDrawer. I would like to merge this and work on smaller follow up PRs.